flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/TestUtils.java

```diff
@@ -18,50 +18,38 @@
 
 package org.apache.flink.tests.util;
 
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.nio.file.Files;
+import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.List;
-import java.util.regex.Pattern;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
 /**
  * General test utilities.
  */
-public enum TestUtils {
-	;
+public class TestUtils {
+
+	private static final String END_TO_END_TESTS_MODULE = "/flink-end-to-end-tests/";
```

Comment:
-1, the current setup via maven was intentionally setup the way it was to make this code independent of the directory structure.

Reply:
OK ,  let me revert back to the previous PR.